core/account.py

```diff
@@ -35,6 +40,102 @@ def get_account(
         raise TypeError("GET account needs at least 1 argument.")
 
 
+def put_account(
+    account_id: str
+) -> Tuple[dict, int]:
+    """put_account Given an account id and an optional role,
```

Comment:
think the comment has a typo with 'optional' here

Reply:
Thanks, good spot. Removed